src/wp-includes/class-walker-nav-menu.php

```diff
@@ -182,6 +182,11 @@ public function start_el( &$output, $data_object, $depth = 0, $args = null, $cur
 		} else {
 			$atts['rel'] = $menu_item->xfn;
 		}
+
+		if ( ! empty( $menu_item->url ) && get_privacy_policy_url() === $menu_item->url ) {
+			$atts['rel'] = empty( $atts['rel'] ) ? 'privacy-policy' : $atts['rel'] . ' privacy-policy';
+		}
+
 		$atts['href']         = ! empty( $menu_item->url ) ? $menu_item->url : '';
```

Comment:
```suggestion
		if ( ! empty( $menu_item->url ) ) {
			if ( get_privacy_policy_url() === $menu_item->url ) {
				$atts['rel'] = empty( $atts['rel'] ) ? 'privacy-policy' : $atts['rel'] . ' privacy-policy';
			}
			$atts['href'] = $menu_item->url;
		} else {
			$atts['href'] = '';
		}

```

@costdev WDYT?

Reply:
@mukeshpanchal27 This makes sense to remove the double `! empty( $menu_item->url )`, thanks!